src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -486,23 +429,17 @@ public void updateSubscribers()
         Set<EntityPlayerMP> oldSubscribers = subscribers;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'oldSubscribers' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)